modules/Bio/EnsEMBL/Variation/DBSQL/CADDAnnotationAdaptor.pm

```diff
@@ -180,9 +180,7 @@ sub new {
 
 sub fetch_by_id {
```

Comment:
Not sure we need to write this as this method is defined in the `BaseAnnotationAdaptor`